storage-hbase/src/main/java/org/apache/kylin/storage/hbase/steps/CubeHFileJob.java

```diff
@@ -101,6 +102,7 @@ public int run(String[] args) throws Exception {
 
             // Automatic config !
             HFileOutputFormat3.configureIncrementalLoad(job, htable);
+            tmpPartitionFilePath = new Path(TotalOrderPartitioner.getPartitionFile(job.getConfiguration()));
```

Comment:
The value get from 'TotalOrderPartitioner.getPartitionFile' will be changed in method 'reconfigurePartitions', are you sure that the value of  'tmpPartitionFilePath' is right?

Reply:
Got it.